dotnet/samples/Demos/ContentSafety/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure AI Content Safety and Prompt Shields service example
+
+[Azure AI Content Safety](https://learn.microsoft.com/en-us/azure/ai-services/content-safety/overview) detects harmful user-generated and AI-generated content in applications and services. Azure AI Content Safety includes text and image APIs that allow to detect material that is harmful.
+
+[Azure AI Content Safety Prompt Shields](https://learn.microsoft.com/en-us/azure/ai-services/content-safety/quickstart-jailbreak) service allows to check your large language model (LLM) inputs for both User Prompt and Document attacks.
+
+Together with Semantic Kernel Prompt Filters, it's possible to define detection logic in dedicated place and avoid mixing it with business logic in applications.
+
+## Prerequisites
+
+1. [OpenAI](https://platform.openai.com/docs/introduction) subscription.
+2. [Azure](https://azure.microsoft.com/free/cognitive-services) subscription.
+3. Once you have your Azure subscription, create a [Content Safety resource](https://aka.ms/acs-create) in the Azure portal to get your key and endpoint. Enter a unique name for your resource, select your subscription, and select a resource group, supported region (East US or West Europe), and supported pricing tier. Then select **Create**.
+4. Create `appsettings.Development.json` file or open existing `appsettings.json` file and set your configuration for `OpenAI` and `AzureContentSafety` sections.
```

Comment:
Can we use the user-secrets approach and encourage that?

Reply:
I've added support for user-secrets, but I think we should support both user-secrets and JSON configuration files. There was some feedback previously that testing with JSON configuration is easier and quicker comparing to user-secrets. It is also aligned with description of this [issue](https://github.com/microsoft/semantic-kernel/issues/4497):
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/e734002a-82d4-48d8-9ccd-5105c10c8b24)